js/react_native/onnxruntime-react-native.podspec

```diff
@@ -19,4 +22,20 @@ Pod::Spec.new do |spec|
 
   spec.dependency "React-Core"
   spec.dependency "onnxruntime-c"
+
+  spec.xcconfig = {
+    'OTHER_CPLUSPLUSFLAGS' => '-Wall -Wextra',
+  }
+
+  if (File.exist?(File.join(root_dir, 'package.json')))
```

Comment:
why do we consider it an error if the root package.json file doesn't exist in build.gradle but not here?

Reply:
added a raise error condition when file does not exist as well

Reply:
Should it be an error? Wondering whether we are potentially making the package completely unusable if it is. 

It seems like the error would only be if the package.json was in an unexpected place. Logging a message and not enabling extensions seems like a better end result than rendering the entire package unusable though. 

e.g. what if their React Native or expo version creates a slightly different directory structure that's still valid and works but results in package.json not being in the hardcoded place we look?

Reply:
ok sure. so sounds like it would be better to log an error message vs. throw an error.

Reply:
updated to logging message now.